scripts/docstrings_checker_test.py

```diff
@@ -84,3 +282,26 @@ def func():
 
         exceptions = docstrings_checker.possible_exc_types(raise_node)
         self.assertEqual(exceptions, set(['Exception']))
+
+    def test_docstringify_with_okey_docstring(self) -> None:
+        valid_docstring = """Docstring that is correctly formated
+            according to the Google Python Style Guide.
+
+            Args:
+                test_value: bool. Just a test argument.
+            """
+        is_valid = isinstance(
+            docstrings_checker.docstringify(valid_docstring),
+            docstrings_checker.GoogleDocstring)
+
+        self.assertEqual(is_valid, True)
+
+    def test_docstringify_with_bad_docstring(self) -> None:
+        invalid_docstring = """Docstring that is incorrectly
+            formated according to the Google Python Style Guide.
+            """
+        is_valid = isinstance(
```

Comment:
Also changed this variable name after the review

Reply:
Right on!